features/multi-column.yml

```diff
@@ -0,0 +1,19 @@
+name: Multi-column layout
+description: Multi-column layout flows an element's content across one or more vertical content areas in a single row, without affecting the `display` properties of its children.
```

Comment:
The bit about "without affecting the `display` properties of its children" is included as a differentiator between mutli-column and grid/flexbox. This is called out in the spec and I've seen it alluded to in posts discussing when to choose this over other options

Reply:
That seems like a good option. I do wonder about using vertical here, as that may differ based on writing mode. Can we use a logical descriptor instead? Oof. `one or more block content areas in a single inline row`?